### ISSUE TITLE
Indent works incurrect after linebreak betwee await and function call

### ISSUE THREAD

**Issue by Alexsey** [eslint/eslint#10251]
**Tell us about your environment**

* **ESLint Version: 4.19.1**
* **Node Version: 8.10.0**
* **npm Version: 5.8.0**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "root": true,
  "globals": {
    "should": false,
    "sinon": false,
    "_": false,
    "angular": false,
    "console": false,
    "inject": false,
    "module": false,
    "window": false
  },
  "env": {
      "browser": true,
      "node": true,
      "es6": true,
      "jquery": true,
      "mocha": true
  },
  "parser": "babel-eslint",
  "parserOptions": {
    "ecmaVersion": 7,
    "ecmaFeatures": {
        "experimentalObjectRestSpread": true
    }
  },
  "rules": {
    "brace-style": [2, "stroustrup", {"allowSingleLine": false}],
    "camelcase": 1,
    "comma-dangle": [1, "never"],
    "curly": 1,
    "dot-notation": 1,
    "eqeqeq": [1, "always", { "null": "ignore" }],
    "indent": [1, 4, {"SwitchCase": 1}],
    "lines-around-comment": [2, {"allowBlockStart": true, "beforeBlockComment": true, "beforeLineComment": true}],
    "new-parens": 1,
    "new-cap": ["error", { "capIsNewExceptions": ["Router"] }],
    "no-unresolved": "2",
    "no-bitwise": 1,
    "no-cond-assign": 1,
    "no-debugger": 1,
    "no-dupe-args": 1,
    "no-dupe-keys": 1,
    "no-empty": 1,
    "no-invalid-regexp": 1,
    "no-invalid-this": 1,
    "no-mixed-spaces-and-tabs": [1, "smart-tabs"],
    "no-multiple-empty-lines": [1, {"max": 2}],
    "no-multi-assign": 1,
    "no-undef": 1,
    "no-eval": 1,
    "no-unreachable": 2,
    "no-unused-vars": [1, { "argsIgnorePattern": "^(res|req|next)$" }],
    "no-loop-func": 1,
    "one-var": [1, { "uninitialized": "never" }],
    "quote-props": [1, "as-needed"],
    "semi": [1, "always"],
    "keyword-spacing": 1,
    "space-unary-ops": [1, {"words": true, "nonwords": false}],
    "space-before-blocks": "error",
    "space-before-function-paren": ["error", {
        "anonymous": "always",
        "named": "never",
        "asyncArrow": "always"
    }],
    "space-in-parens": ["error", "never"],
    "vars-on-top": 1,
    "wrap-iife": [1, "outside"],
    "yoda": [1, "never"],
    "quotes": ["error", "single", { "avoidEscape": true }],

    //ES6 Stuff
    "arrow-parens": ["error", "as-needed"],
    "arrow-spacing": 1,
    "constructor-super": 1,
    "no-class-assign": 1,
    "no-const-assign": 1,
    "no-dupe-class-members": 1,
    "no-this-before-super": 1,
    "no-var": 1,
    "object-shorthand": 1,
    "prefer-arrow-callback": 1,
    "prefer-const": 1,
    "prefer-destructuring": 1,
    "prefer-template": 2,
    "template-curly-spacing": ["error", "never"],
    "no-return-await": 2,
    "no-restricted-syntax": [
        "error",
        {
        "selector": "FunctionDeclaration[async=false][id.name=/Async(Mock){0,1}$/]",
        "message": "Function ending in 'Async' must be declared async"
        },
        {
        "selector": "FunctionDeclaration[async=true][id.name!=/Async(Mock){0,1}$/]",
        "message": "Async function name must end in 'Async'"
        },
        {
        "selector": "MethodDefinition[value.async=false][key.name=/Async(Mock){0,1}$/]",
        "message": "Method ending in 'Async' must be declared async"
        },
        {
        "selector": "MethodDefinition[value.async=true][key.name!=/Async(Mock){0,1}$/]",
        "message": "Async method name must end in 'Async'"
        },
        {
        "selector": "Property[value.type=/FunctionExpression$/][value.async=false][key.name=/Async(Mock){0,1}$/]",
        "message": "Function ending in 'Async' must be declared async"
        },
        {
        "selector": "Property[value.type=/FunctionExpression$/][value.async=true][key.name!=/Async(Mock){0,1}$/]",
        "message": "Async function name must end in 'Async'"
        },
        {
        "selector": "VariableDeclarator[init.type=/FunctionExpression$/][init.async=false][id.name=/Async(Mock){0,1}$/]",
        "message": "Function ending in 'Async' must be declared async"
        },
        {
        "selector": "VariableDeclarator[init.type=/FunctionExpression$/][init.async=true][id.name!=/Async(Mock){0,1}$/]",
        "message": "Async function name must end in 'Async'"
        },
        {
        "selector": "*:not(ClassDeclaration) ExpressionStatement > CallExpression > MemberExpression > Identifier[name=/^_.+/]",
        "message": "Don't call private methods ( e.g. _PrivateMethod() ) on an instaniated class"
        }
    ],
    "no-warning-comments": ["error", { "terms": ["todo", "fixme", "broken", "hack"], "location": "anywhere" }]
  }
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
;(async () => {
    const a = await
        f() // <- error is here
})()

async function f () {
    return 2
}
```

**What did you expect to happen?**

No indent error at line 3. Just like it would be without `await`

**What actually happened? Please include the actual, raw output from ESLint.**
`ESLint: Expected indentation of 4 spaces but found 8. (indent)`
